PyCut.py

```diff
@@ -2,10 +2,12 @@
 import pygame
 from game import PyCutGame as PyCut
 import cProfile
+import gi
+from gi.repository import Gdk
 
 def main():
     pygame.init()
-    pygame.display.set_mode((1200, 900), pygame.RESIZABLE)
+    pygame.display.set_mode((Gdk.Screen.width(), Gdk.Screen.height()), pygame.RESIZABLE)
```

Comment:
Passing (0, 0) will use the full screen width, no need to use Gdk here.

Reply:
Wow, That is a new info for me!!